requirements-base.txt

```diff
@@ -102,4 +102,4 @@ phabricator>=0.7.0
 # test dependencies, but unable to move to requirements-test until
 # sentry.utils.pytest and sentry.testutils are moved to tests/
 selenium>=4.1.5
-sqlparse>=0.2.4
+sqlparse>=0.3.0
```

Comment:
Don't we have to import this as well ?
https://github.com/googleapis/python-spanner-django

Reply:
yes but i couldn't figure out how to add a git dependency in our requirements. We need a fix from master to get anywhere at all